btr-web/btr-main-app/components/individual-person/AddNew.vue

```diff
@@ -271,6 +304,14 @@ const taxInfoModel = computed({
 
 // tax residency
 const isTaxResident: Ref<string | undefined> = ref(undefined)
+
+const isUnableToObtainOrConfirmInformation = ref(false)
+const isUnableToObtainOrConfirmInformationDetails = ref('')
+const isUnableToObtainOrConfirmInformationDetailsChange = () => {
```

Comment:
if you convert this to a watcher on `isUnableToObtainOrConfirmInformationDetails` it'll have the desired effect. Double check with Scott, but I think it should be getting checked / unchecked in real time not blur?